tests/benchmarks/http-benchmarks/README.md

```diff
@@ -0,0 +1,33 @@
+# HTTP Client Engine Benchmarks
+
+This project contains benchmarks for the [HTTP engine implementations](../../../runtime/protocol/http-client-engines).
+
+## Testing
+
+```sh
+./gradlew :tests:benchmarks:http-benchmarks:benchmark
+```
+
+Baseline `0.10.3-SNAPSHOT` on EC2 **[m5.4xlarge](https://aws.amazon.com/ec2/instance-types/m5/)** with **Corretto-11.0.15.9.1**:
+
+The download/upload throughput benchmarks are an approximation of how much data in MB/s we are able to process.
+
+```
+jvm summary:
+Benchmark                                      (httpClientName)   Mode  Cnt      Score     Error  Units
+HttpEngineBenchmarks.downloadThroughputNoTls             OkHttp  thrpt    5    129.300 ±   1.796  ops/s
```

Comment:
I haven't been able to explain the difference between `OkHttp` download throughput and `Ktor_OkHttp`. Theoretically these should be much closer given they are the same underlying engine but on the m5 instance they are not. Locally when I run the benchmark they are closer together (with `OkHttp` sometimes barely beating `Ktor_OkHttp`). 


Also FYI the benchmark throughput here is a rough estimation of `MB/s` but I can't figure out how to change the Units in `jmh`/`kotlinx-benchmark`.